packs/octopusdeploy/actions/deploy_release.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: deploy_release
+runner_type: run-python
+description: Deploy a release in Octopus.
+enabled: true
+entry_point: deploy_release.py
+parameters:
+  release_id:
+    type: string
+    description: ID of the release in Octopus
+    required: true
+    position: 0
```

Comment:
Positions are used only for script actions that accept positional arguments. You don't need this for python actions.